common/predictive-text/README.md

```diff
@@ -1,408 +1,61 @@
-Keyman/Language Modelling layer 
-================================
+Language Modelling Layer (LMLayer)
```

Comment:
It's not obvious from the GitHub's diff view, but I added a new README that briefly explains the two-stage build process, and links to the original README.